.github/workflows/c_actions.yml

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2020 The University of Manchester
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+# This workflow will install Python dependencies, run tests, lint and rat with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: C Actions
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Checkout SupportScripts
+      uses: actions/checkout@v2
+      with:
+        repository: SpiNNakerManchester/SupportScripts
+        path: support
+        
+    - name: Set SPINN_DIRS
+      run: |
+        echo "SPINN_DIRS=$PWD" >> $GITHUB_ENV
+        
+    - name: Set other C environment variables
+      run: |
+        echo "SPINN_PATH=$SPINN_DIRS/tools/boot" >> $GITHUB_ENV
+        echo "$SPINN_DIRS/tools" >> $GITHUB_PATH
+        echo "PERL5LIB=$SPINN_DIRS/tools" >> $GITHUB_ENV
+        echo "GCC_COLORS=error=01;31:warning=01;35:note=01;36:range1=32:range2=34:locus=01:quote=01:fixit-insert=32:fixit-delete=31:diff-filename=01:diff-hunk=32:diff-delete=31:diff-insert=32" >> $GITHUB_ENV
```

Comment:
This environment variable can be put as an `env:` on the “Build C code” step; that's the only place where it is needed and it is not path-dependent.

Reply:
Environment variable moved.